pkg/apis/eks.cattle.io/v1/types.go

```diff
@@ -52,26 +52,40 @@ type EKSClusterConfigSpec struct {
 }
 
 type EKSClusterConfigStatus struct {
-	Phase          string   `json:"phase"`
-	VirtualNetwork string   `json:"virtualNetwork"`
-	Subnets        []string `json:"subnets"`
-	SecurityGroups []string `json:"securityGroups"`
+	Phase                    string    `json:"phase"`
+	VirtualNetwork           string    `json:"virtualNetwork"`
+	Subnets                  []string  `json:"subnets"`
+	SecurityGroups           []string  `json:"securityGroups"`
+	LaunchTemplateID         string    `json:"launchTemplateID"`
```

Comment:
Can you name this `generatedLaunchTemplateID` or `managedLaunchTemplateID` to clarify that is the only time it is set?

Reply:
Done.